src/App.js

```diff
@@ -2,7 +2,9 @@ import MyCulc from './components/Calculator';
 
 function App() {
   return (
-    <MyCulc />
+    <>
```

Comment:
- [ ] One more thing, Your issue page is not placed correctly in readme and it's not leading me to the right place, which means it's broken. Kindly fix it as well.

 

<img width="536" alt="Screenshot 2023-05-04 at 23 02 26" src="https://user-images.githubusercontent.com/69011963/236290368-8588e35f-ba60-4c21-806d-1531232ae745.png">

Reply:
Ok